dev/deps/spark-deps-hadoop-3.2-hive-2.3

```diff
@@ -166,14 +167,31 @@ kerby-pkix/1.0.1//kerby-pkix-1.0.1.jar
 kerby-util/1.0.1//kerby-util-1.0.1.jar
 kerby-xdr/1.0.1//kerby-xdr-1.0.1.jar
 kryo-shaded/4.0.2//kryo-shaded-4.0.2.jar
-kubernetes-client/4.9.2//kubernetes-client-4.9.2.jar
-kubernetes-model-common/4.9.2//kubernetes-model-common-4.9.2.jar
-kubernetes-model/4.9.2//kubernetes-model-4.9.2.jar
+kubernetes-client/4.10.3//kubernetes-client-4.10.3.jar
+kubernetes-model-admissionregistration/4.10.3//kubernetes-model-admissionregistration-4.10.3.jar
+kubernetes-model-apiextensions/4.10.3//kubernetes-model-apiextensions-4.10.3.jar
+kubernetes-model-apps/4.10.3//kubernetes-model-apps-4.10.3.jar
+kubernetes-model-autoscaling/4.10.3//kubernetes-model-autoscaling-4.10.3.jar
```

Comment:
Are all of these additional deps needed - if they really aren't we might exclude them for tidiness.

Reply:
I don't think we have a choice here, it looks like kubernetes-client split their jar into smaller dependencies:
https://github.com/fabric8io/kubernetes-client/issues/2108, and since we depend on the top level `kubernetes-client`, we must pull these in.